platform/test/spec/testActivityConfigValidatorSpec.js

```diff
@@ -0,0 +1,84 @@
+/*global describe, it, expect, beforeEach --  functions provided by Jasmine */
+import { ActivityConfigValidator } from "../../src/ActivityConfigValidator.js"
+import { ACTIVITY_2PANELS_1ACTION } from "../resources/TestActivityFiles.js";
+import { ConfigValidationError } from "../../src/ConfigValidationError.js"
+
+
+const EXPECTED_FILE_TYPE = "ActivityConfig"; 
+
+describe("ActivityConfigValidator", () => {
```

Comment:
Is this really testing all error conditions that the configuration validator checks for?

Reply:
No its schema validation, there is another validator ActivityValidator that does more of the semantic checking that has its own tests in #190.